events/2021.09.Online/Discovery/README.md

```diff
@@ -1 +1,227 @@
+                  "description":"Success response",
+                  "htv:statusCodeValue":204
+               },
+               "additionalResponses":[
+                  {
+                     "description":"TD with the given id not found",
+                     "contentType":"application/problem+json",
+                     "htv:statusCodeValue":404
+                  }
+               ],
+               "scopes":"write"
+            }
+         ]
+      }
+   },
+   "properties":{
+      "retrieveTD":{
+         "description":"Retrieve a Thing Description",
+         "uriVariables":{
+            "id":{
+               "title":"Thing Description ID",
+               "type":"string",
+               "format":"iri-reference"
+            }
+         },
+         "forms":[
+            {
+               "href":"/td/{id}",
+               "htv:methodName":"GET",
+               "response":{
+                  "description":"Success response",
+                  "htv:statusCodeValue":200,
+                  "contentType":"application/td+json"
+               },
+               "additionalResponses":[
+                  {
+                     "description":"TD with the given id not found",
+                     "contentType":"application/problem+json",
+                     "htv:statusCodeValue":404
+                  }
+               ],
+               "scopes":"read"
+            }
+         ]
+      },
+      "retrieveTDs":{
+         "description":"Retrieve all Thing Descriptions",
+         "forms":[
+            {
+               "href":"/td",
+               "htv:methodName":"GET",
+               "response":{
+                  "description":"Success response",
+                  "htv:statusCodeValue":200,
+                  "contentType":"application/ld+json"
+               },
+               "scopes":"readAll"
+            }
+         ]
+      },
+      "searchSPARQL":{
+         "description":"SPARQL semantic search",
+         "uriVariables":{
+            "query":{
+               "title":"A valid SPARQL 1.1. query",
+               "type":"string"
+            }
+         },
+         "forms":[
+            {
+               "href":"/search/sparql?query={query}",
+               "htv:methodName":"GET",
+               "response":{
+                  "description":"Success response",
+                  "htv:statusCodeValue":200
+               },
+               "additionalResponses":[
+                  {
+                     "description":"Query not provided or contains syntax errors",
+                     "contentType":"application/problem+json",
+                     "htv:statusCodeValue":400
+                  }
+               ],
+               "scopes":"search"
+            },
+            {
+               "href":"/search/sparql",
+               "htv:methodName":"POST",
+               "response":{
+                  "description":"Success response",
+                  "contentType":"application/json",
```

Comment:
Should be `application/sparql-results+json`?

[Discovery document](https://w3c.github.io/wot-discovery/#search-semantic) says that "A successful response MUST have 200 (OK) status, and depending on the type of query contain by default as Content-Type header application/ld+json for CONSTRUCT and DESCRIBE queries or application/json for SELECT or ASK."
But [W3C Recommendation (SPARQL 1.1 Query Results JSON Format)](https://www.w3.org/TR/sparql11-results-json/#content-type)
suggests to use "application/sparql-results+json" and also your implementation returns "content-type: application/sparql-results+json".

Reply:
Thanks @k-toumura for the feeback!

You are right and I'll correct that. Looks like we made copy&paste without verification since it is also wrong in the sample TD of the Discovery spec ;-)